k8s/utils/health/test_health.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+import cluster_health_check
+from mock import patch, Mock
+
+class Test_Object(object):
+    pass 
+ 
+class TestHealthCheck(unittest.TestCase):
+     
+    def setUp(self):
+        self.node_count = 3
+        pass
+ 
+    def get_node_list_with_conditions(): 
+        test_object = Test_Object()
+        item_array = [Test_Object() for i in range(self.node_count)]
+        conditions = ["Ready", "Not Ready", "Maybe"]
+        for item in item_array:
+            item.status = Test_Object()
+            item.status.conditions = []
+            for condition in conditions:
+                new_condition = Test_Object()
+                new_condition.type = condition
+                item.status.conditions.append(new_condition)
+        test_object.items = item_array                      
+        return test_object
+        
+    def test_create_api(self):
+        self.assertNotEqual(cluster_health_check.create_api(), None)
+        
+    @patch('cluster_health_check.client.CoreV1Api')
+    def test_get_nodes(self, v1_mock):
+        test_object = Test_Object()
+        test_object.items = [i for i in range(len(self.node_count))]        
```

Comment:
what is the use or `test_object.items` can we just use `range` to populate it?

Reply:
Items are the list of nodes that we are mocking here with a dummy list to check the code flow. Since only a list is expected here we can populate it with range here.